### ISSUE TITLE
Does not compile in xcode 6.4 (6e35b)

### ISSUE THREAD

**Issue by SleepyBrett** [svdo/ipatool#2]
XCode surfaces 20 issues. Many of them are similar to 'AnyObject?' is not convertible to 'SOMETYPE'; did you mean to use 'as!' to force downcast.

**Comment by SleepyBrett** [comment]
After making a number of fixes to a language I don't really know, I managed to make it compile. However now I'm having some issues processing a commandline that looks like this.

ipatool xxx.ipa resign xxx.mobileprovision com.blah.foo.xxx

I get 'Need parameters for ipa path and new provisioning profile'.

Which I trace to validateArgs in IPTCommandResign.swift, other digging in that function seems to imply that 'resign' should have been removed from the args array (it tests args members 0 and 1 to make sure they are readable files.. however 1 at this point is 'resign').

This leads me to convertArgsForCompatibility, where .. frankly I don't understand this at all..starting with

```
    if (args.count >= 2 &&
        (args[1] == "provisioning-profile" || args[1] == "bundle-identifier")) {
```

Is this some holdover from a previous incarnation?
